js/popupGallery.js

```diff
@@ -0,0 +1,74 @@
+import {picturesContainer} from './renderingPicture.js';
+
+const displayingWindow = document.querySelector('.big-picture');
+const closeBigPhoto = displayingWindow.querySelector('.big-picture__cancel');
+const bigPicture = displayingWindow.querySelector('.big-picture__img img');
+const likesCounter = displayingWindow.querySelector('.likes-count');
+const commentCount = displayingWindow.querySelector('.comments-count');
+const commentsList = displayingWindow.querySelectorAll('.social__comments li');
+const descriptionPhoto = displayingWindow.querySelector('.social__caption');
+
+const addClass = ()=>{
+  const socialCommentCount = displayingWindow.querySelector('.social__comment-count');
+  const commentsLoader = displayingWindow.querySelector('.comments-loader');
+  document.querySelector('body').classList.add('modal-open');
+  socialCommentCount.classList.add('hidden');
+  commentsLoader.classList.add('hidden');
+};
+
+const closePopup = ()=>{
```

Comment:
Если `showPopup` то лучше назвать `hidePopup`  show/hide, close/open
```suggestion
const hidePopup = (e) => {
  if (e.key && e.key !== 'Escape') {
    return;
  }
  e.preventDefault();
  displayingWindow.classList.add('hidden');
  closeBigPhoto.removeEventListener('click', hidePopup);
  document.removeEventListener('keydown', hidePopup);
  document.body.classList.remove('modal-open');
};
```
И с описанием, возможно, придется назвать и hidePopupHandler, но не суть
```ts
const hidePopup = (e) => {
 Проверяем, был ли ивент вызван нажатием на кнопку и если был и это не esc, то ничего не делаем
  if (e.key && e.key !== 'Escape') {
    return;
  }
  // Отменяем стандарное поведение
  e.preventDefault();
  // Скрываем попап
  displayingWindow.classList.add('hidden');
  // Убераем ивент на кнопку 
  closeBigPhoto.removeEventListener('click', hidePopup);
  // Убираем ивент на закрытие по esc
  document.removeEventListener('keydown', hidePopup);
  // Убираем класс открытого попапа у body 
  document.body.classList.remove('modal-open');
};
```